pkg/k8s/client.go

```diff
@@ -468,3 +489,99 @@ func (c *Client) GetMeshConfig() configv1alpha2.MeshConfig {
 func (c *Client) GetOSMNamespace() string {
 	return c.osmNamespace
 }
+
+func (c *Client) ListEgressPolicies() []*policyv1alpha1.Egress {
+	var policies []*policyv1alpha1.Egress
+
+	for _, egressIface := range c.informers.List(osminformers.InformerKeyEgress) {
+		egressPolicy := egressIface.(*policyv1alpha1.Egress)
+
+		if !c.IsMonitoredNamespace(egressPolicy.Namespace) {
+			continue
+		}
+		policies = append(policies, egressPolicy)
+	}
+
+	return policies
+}
+
+// GetIngressBackendPolicy returns the IngressBackend policy for the given backend MeshService
+func (c *Client) GetIngressBackendPolicy(namespace, svcName string, targetPort int) *policyv1alpha1.IngressBackend {
```

Comment:
I would just remove this get.. technically we don't really have a get, so the get can be moved up into the compute/client, ie:


```
GetIngressBackendPolicyByService -> calls k8s.ListIngressBackendPolicy, and filters appropriately.
```

The rule of thumb is that Gets should be by just name/namespace. Any other fields are a filter on a list